src/components/editor/document-bar/document-info/document-info-time-line.tsx

```diff
@@ -19,16 +19,16 @@ export enum DocumentInfoLineWithTimeMode {
   EDITED
 }
 
-export const DocumentInfoTimeLine: React.FC<DocumentInfoLineWithTimeProps> = ({ time, mode, userName, profileImageSrc }) => {
+export const DocumentInfoTimeLine: React.FC<DocumentInfoLineWithTimeProps> = ({ time, mode, userName, profileImageSrc, size }) => {
   useTranslation()
 
   const i18nKey = mode === DocumentInfoLineWithTimeMode.CREATED ? 'editor.modal.documentInfo.created' : 'editor.modal.documentInfo.edited'
   const icon: IconName = mode === DocumentInfoLineWithTimeMode.CREATED ? 'plus' : 'pencil'
 
   return (
-    <DocumentInfoLine icon={icon}>
+    <DocumentInfoLine icon={icon} size={size}>
       <Trans i18nKey={i18nKey} >
-        <UserAvatar photo={profileImageSrc} additionalClasses={'document-info-avatar font-style-normal bold font-weight-bold'} name={userName}/>
+        <UserAvatar photo={profileImageSrc} additionalClasses={`font-style-normal bold font-weight-bold ${size ? 'document-info-avatar' : ''}`} name={userName} size={size ? 'lg' : undefined}/>
```

Comment:
Is the condition for `document-info-avatar` still needed? You deleted the scss file that contained this rule.

Reply:
Seems like a mistake, I check it

Reply:
Removed